WordPress/src/main/res/layout/new_site_creation_domains_item.xml

```diff
@@ -18,7 +18,10 @@
         android:clickable="false"
         android:focusable="false"
         android:focusableInTouchMode="false"
+        android:paddingBottom="@dimen/margin_medium"
+        android:paddingEnd="@dimen/margin_large"
```

Comment:
```suggestion
        android:paddingEnd="@dimen/margin_extra_large"
```

Shouldn't this be `margin_extra_large` for the end padding to match 16dp from @SylvesterWilmott's [comment](https://github.com/wordpress-mobile/WordPress-Android/issues/8832#issuecomment-449345909)?

Reply:
I missed that comment, thanks for noticing @theck13!

Reply:
When you apply the suggested commit it's authored by both, really like that! And thank you for the suggestion line as well @theck13, it saved me a bunch of time from switching branches :)

Reply:
Ahh, nice!  Clever feature.  No problem.  Thanks for the fix!